invenio_oauthclient/models.py

```diff
@@ -30,7 +30,7 @@
 from sqlalchemy.ext.mutable import MutableDict
 from sqlalchemy.orm import backref
 from sqlalchemy_utils import JSONType
-from sqlalchemy_utils.types.encrypted import EncryptedType
+from sqlalchemy_utils.types.encrypted.encrypted_type import EncryptedType
```

Comment:
```python
try:
    # SQLAlchemy-Utils <=0.32
    from sqlalchemy_utils.types.encrypted import EncryptedType
except ImportError:
    # SQLAlchemy-Utils >=0.33
    from sqlalchemy_utils.types.encrypted.encrypted_type import EncryptedType
```

In order to not force existing installations to upgrade SQLAlchemy-Utils immediately, I would go with above. ``EncryptedType`` is being used in other modules (e.g. OAuth2Server I think), hence if not all modules are fixed and released at the same time then it will cause conflicts

Reply:
Actually, on second look into SQLAlchemy-Utils, following should work on both versions:

```
from sqlalchemy_utils import EncryptedType
```